x-pack/plugins/cases/public/components/markdown_editor/renderer.tsx

```diff
@@ -16,16 +16,25 @@ interface Props {
   disableLinks?: boolean;
 }
 
+const withDisabledLinks = (disableLinks?: boolean): React.FC<EuiLinkAnchorProps> => {
```

Comment:
Create a HOC that returns a `MarkdownLink` component with `disableLinks` so we can declare a `displayName`.